moveit_core/planning_scene/CMakeLists.txt

```diff
@@ -1,7 +1,8 @@
 set(MOVEIT_LIB_NAME moveit_planning_scene)
 
 add_library(${MOVEIT_LIB_NAME} src/planning_scene.cpp)
-set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
+#TODO: Fix the versioning
+# set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
```

Comment:
This was added in https://github.com/ros-planning/moveit/pull/273. Maybe @v4hn can confirm this but perhaps we can take the same approach as the [urdf_parser](https://github.com/ros/urdfdom/blob/3ffc4eebb22e24bef503849eb43719a7c148e999/urdf_parser/CMakeLists.txt#L5)